src/server/rdb_save.cc

```diff
@@ -21,13 +22,17 @@ extern "C" {
 #include "redis/zset.h"
 }
 
+#include "base/flags.h"
 #include "base/logging.h"
 #include "server/engine_shard_set.h"
 #include "server/error.h"
 #include "server/rdb_extensions.h"
 #include "server/snapshot.h"
 #include "util/fibers/simple_channel.h"
 
+ABSL_FLAG(bool, save_enhanced_compression, true, "DF save compressed blobs of multiple entries");
```

Comment:
there is a bug when you run dragonfly with `save_enhanced_compression=false` it does not compress strings with lzf as it did beforehand.

Reply:
Owww not only that but I do double compression lzf and zstd when I have the flag=true